src/common/utils.ts

```diff
@@ -13,3 +13,15 @@ export function join(...paths: string[]) {
     .replace(/\/+/g, '/'); // Replace multiple slashes with a single slash
   
 }
+
+export function isURLSearchParams(val: object) {
+  return typeof URLSearchParams !== 'undefined' && val instanceof URLSearchParams;
```

Comment:
Why this syntax is inserted? 
```ts
typeof URLSearchParams !== 'undefined'
```

Reply:
for browser

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#browser_compatibility
its from web workers feature and used by other browsers, I see that users use axios-logger for browser too (not only for node.js)